src/map/pc.cpp

```diff
@@ -6259,17 +6259,6 @@ bool pc_isUseitem(map_session_data *sd,int n)
 	if( itemdb_group.item_exists(IG_MERCENARY, nameid) && sd->md != nullptr )
 		return false; // Mercenary Scrolls
 
-	if( item->flag.group || item->type == IT_CASH) {	//safe check type cash disappear when overweight [Napster]
```

Comment:
You removed the cash item check.
Intentional?

Reply:
Yeah, because of where I moved it to, is no longer necessary.